entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -25,28 +25,58 @@ It has not yet shipped in any 3.0 preview.
 
 **Old behavior**
 
-Prior to EF Core 3.0, any part of a query that could not be translated to SQL would be instead evaluated on the client.
-Client evaluation of potentially expensive expressions only triggered a warning by default.
+Before 3.0, when EF Core failed to convert a query expression to SQL or to a parameter, it would instead evaluate it on the client.
+By default, client evaluation of potentially expensive expressions only triggered a warning.
 
 **New behavior**
 
-Starting with EF Core 3.0, we only allow the top level projection (the expression passed to the last Select LINQ operator in the query) to be evaluated on the client.
-If expressions in any other part of the query cannot be either captured as a parameter or translated to SQL, then an exception is thrown.
+Starting with 3.0, EF Core only allows expressions in the top-level projection (the last `Select()` in the query) to be evaluated on the client.
+If expressions in any other part of the query can't be converted to either parameters or SQL, an exception is thrown.
 
 **Why**
 
-The automatic client-evaluation of queries allowed many queries to be executed even if important parts of them could not be translated.
-This can result in unexpected and potentially damaging behavior that may only become apparent in production.
-For example, a query containing an expression in the Where LINQ operator that cannot be translated, can result in all rows from the table to be transferred from the database server, and the filter to be applied on the client.
-This can easily go undetected if the table contains only a few rows in development and staging, but hit hard when you move the application to production where the table contains millions of rows.
+Automatic client evaluation of queries allows many queries to be executed even if important parts of them can't be translated.
+This behavior can result in unexpected and potentially damaging behavior that may only become evident in production.
+For example, a query containing a condition in `Where()` which can't be translated, results in all rows from the table being transferred from the database server, and the filter being applied on the client.
+This situation can easily go undetected if the table contains only a few rows in development.
+But once in production, if the table contains millions of rows, it can be enough to take down the application. 
```

Comment:
This sentence split feels clumsy.

Reply:
@ajcvickers Are you referring to the formatting or the contents of the sentence?

Reply:
The contents of the last two sentences. Specifically, the wording of the new sentence that starts with "But".